app/src/components/Statistics/Statistics.js

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect, useState } from "react";
+import { StatisticsElement } from "./StatisticsElement";
+import { StatisticsData } from "./StatisticsData";
```

Comment:
StatisticsData should be lowercase. this is not a react component. It's just some data object :)

Reply:
changed.